### DISCUSSION TITLE
Monitor does not turn on anymore, black screen

### DISCUSSION THREAD

**Discussion by ozanmikael** [xanderfrangos/twinkle-tray#646] [Q&A]
Hi,

I've been using TwinkleTray for a very long time now and never had issues.
Back then, a couple of months ago, I turned off every monitor with the "turn off Monitors" option inside of TwinkleTray:

![image](https://github.com/xanderfrangos/twinkle-tray/assets/58893898/318c2f4a-68b6-478b-955b-c0e10cd27abd)

One of my monitors didnt turn on anymore and I do not remember how, but I fixed it somehow and it worked again.

Now.. yesterday I did the same. I turned my monitor screens off with that option and one of my monitors are not turning back on anymore.

I tried so much different stuff but I am just not able to turn it back on again and its really frustrating.
The screen is just black and nothing works.

The monitor itself works with different inputs but I cant use it with my PC anymore.

Is there anything I could do to solve this issue?

**Comment by xanderfrangos** [comment]
Closing due to being a duplicate of https://github.com/xanderfrangos/twinkle-tray/issues/647, which has been resolved.
